libs/components/progress-indicator/src/lib/modules/progress-indicator/types/progress-indicator-nav-button-type.ts

```diff
@@ -1,5 +1,5 @@
 /**
- * Specifies the nav button type. The valid options are `finish`, `next`, `previous`, and `reset`.
+ * The nav button type. The valid options are `finish`, `next`, `previous`, and `reset`.
```

Comment:
```suggestion
 * The nav button type.
```
Listed under Type aliases.